fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1754,8 +1743,10 @@ struct RenameTenantImpl {
 		// Erase the tenant entry itself
 		ManagementClusterMetadata::tenantMetadata().tenantMap.erase(tr, self->oldName);
 
-		// Unlike DeleteTenantImpl, we do not decrement the tenant count because a rename should
-		// preserve the total number of tenants.
+		// Remove old tenant from tenant count
+		ManagementClusterMetadata::tenantMetadata().tenantCount.atomicOp(tr, -1, MutationRef::AddValue);
+		ManagementClusterMetadata::clusterTenantCount.atomicOp(
+		    tr, tenantEntry.assignedCluster.get(), -1, MutationRef::AddValue);
```

Comment:
I haven't checked now, is this function still different from the one in the delete impl?

Reply:
Yes because the delete impl doesn't take the tenant entry as a parameter and has to do a double delete if there is a pair